src/lib/style/theme.js

```diff
@@ -34,6 +34,32 @@ export const grayLightest = '#E9EBED';
 
 // Export the default light & dark theme
 export const defaultTheme = {
+  dark: {
+    danger: '#E84855',
+    warning: '#F8F32B',
+    healthyLight: '#0AADA6',
```

Comment:
I am just thinking since we will degrade the `healthy` color, shall we rename `healthLight` to `healthy` here?

Reply:
Yes, you're right.